### ISSUE TITLE
Nothing works until you refresh the page (Ajax issue)

### ISSUE THREAD

**Issue by fregante** [refined-github/refined-github#5719]
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.
- [X] The bug happens after clearing extension cache. The "Clear cache" button below can also be found at the bottom of the options page.

### Example URLs

Everywhere

### Extension version

22.5.22

### Browser(s) used

Safari

### Description

I keep noticing more and more features are missing until I refresh the page. Example:

1. Visit https://github.com/refined-github/refined-github
2. Open a folder
3. Notice lack of `ci-link`, `rgh-linkify-features`, `quick-file-edit`,  `clean-repo-tabs`, etc

https://user-images.githubusercontent.com/1402241/175481173-960b33f1-f928-43d7-ab5e-2a594c84e261.mov

Interestingly I found the `New Issue` textareas to not turn into rich text editors sometimes (GitHub issue)

![before](https://user-images.githubusercontent.com/1402241/175481415-5f913688-a521-4780-a35f-a9f112c41350.gif)

### Include in this issue:

- [X] Screenshots/video/gif demonstrating the bug, if it’s visual
- [ ] Console errors, if any

**Comment by fregante** [comment]
I'm pretty sure they changed the ajaxed event because `log-features` doesn't show absolutely any features after navigation

**Comment by fregante** [comment]
Indeed new event names:

```
turbo:before-cache
turbo:before-fetch-request
turbo:before-fetch-response
turbo:before-render
turbo:before-stream-render
turbo:before-visit
turbo:click
turbo:frame-load
turbo:frame-render
turbo:load
turbo:reload
turbo:render
turbo:submit-end
turbo:submit-start
turbo:visit
```

**Comment by fregante** [comment]
I saw this issue in Chrome, but it's not for everything, just for CSS-in-JS features like `clean-pinned-issues`. This is because `turbo` is cleaning the classes on body.

An easy fix would be to just add the classes to `document.documentElement` instead, making sure we don't have any `body.rgh-whatever-class` selectors anywhere.

<img width="840" alt="Screen Shot 13" src="https://user-images.githubusercontent.com/1402241/176480180-8c9b992f-1ca7-4540-aebd-c5286d32fbd7.png">

**Comment by fregante** [comment]
😩

**Comment by gamemaker1** [comment]
I'm seeing #5738 in the `useful-not-found-page` too: https://github.com/Verifiable-Presentation/impl/blob/main/modules/registry/source/handlers/presentation.ts.

It acts very weird - sometimes on first load the visualized path is seen only once, and then if you reload it it shows up twice, and vice versa.

**Comment by fregante** [comment]
Reported in #5733, fixed in #5742, a new release is coming today, I'm looking into https://github.com/refined-github/refined-github/issues/5751 right now

**Comment by gamemaker1** [comment]
Oops I somehow missed #5733, sorry about that. Thanks for the quick fixes!

**Comment by ian-h-chamberlain** [comment]
Is this possibly the same cause of custom CSS not applying until after a refresh? Example steps:

* Add custom CSS rule to `refined-github` preferences, e.g. 
  ```css
	.blob-code { background-color: #ff0000 !important; }
  ```
* Click link e.g. #5752 
* Click "files changed" tab
* Refresh to see CSS rules applied to page

Should I file a new issue for that, or is it just a lingering side effect of the same issue that happened here?

**Comment by fregante** [comment]
The CSS you add to the preferences is applied on the next full page load (or refresh), it’s not “live” like it would with a dedicated extension (like Stylish). That’s working as intended

**Comment by fregante** [comment]
hmmm you might be onto something, I'm having issues with this in 22.7.1 on Safari. Can you open a new issue?

**Comment by 13steinj** [comment]
Just a heads up, having the same issue on a GitHub Enterprise 3.5.4; maybe related to #4695. Though this may be because the extension assumes newer selectors that old versions of Github don't have.

I'm well aware that 3.5.4 is now EOL, but this was broken even when it wasn't.
My org is expected to update to 3.7 shortly, and then asap to 3.9 & 3.10. I'll report back if this is broken on a 3.7 instance; but I'll probably have to heavily redact any screencaptures.

Probably related note; if I turn on debugging features on, I see this warning on every load:

```
Refined GitHub has been loaded twice. This may be because:

• You loaded the developer version, or
• The extension just updated

If you see this at every load, please open an issue mentioning the browser you're using and the URL where this appears.
```

This does not occur on the main www.github.com

**Comment by fregante** [comment]
DO NOT post on old issues. This is clear in the banner. Your comment is unrelated too.
